docs/quickstart.rst

```diff
@@ -4,11 +4,39 @@ Quickstart
 
 Just want to get a name and make it resolve to something? Here's how.
 
-First, download `ensutils-testnet.js`_ to your local machine, and import it into an Ethereum console on a node synced to ropsten or rinkeby:
+First, make sure your client is in **sync with a network** (mainnet, ropsten, rinkeby, etc.). You can use eth.syncing in your Ethereum console to track progress.
+
+You can utilize geth: https://github.com/ethereum/go-ethereum to connect a network and begin the process of syncing the chain.
+
+Next, start an Ethereum console:
+
+::
+
+    Mainnet: geth attach
+    Ropsten: geth --testnet attach
+    Rinkeby: geth --rinkeby attach
+
+Download `ensutils-testnet.js`_ to your local machine, and import it into an Ethereum console on a node synced to ropsten or rinkeby:
 
 ::
 
-    loadScript('/path/to/ensutils-testnet.js');
+    loadScript('/path/to/ensutils-testnet.js'); // This will connect to the ropsten testnet
+
+If you want to use Rinkeby, you'll need to change in ensutils-testnet.js:
+
+::
+
+        contract address: 0xe7410170f87102df0055eb195163a03b7f2bff4a (line 220)
+        publicResolver address: 0x5d20cf83cb385e06d2f2a892f9322cd4933eacdc (line 1314)
+
+If you need to unlock your account to execute certain transactions:
+
+::
+
+    // Careful as this will record your password in your history.
```

Comment:
It shouldn't - geth doesn't store any calls to web3.personal in the command history in the console.

Reply:
Yes, learned this later...